roles/parse_operator_bundle/tasks/main.yml

```diff
@@ -0,0 +1,97 @@
+---
+- name: "Inspect the bundle image with skopeo"
+  shell: "skopeo inspect docker://{{ bundle_image }}"
+  register: skopeo_inspect_result
+
+- name: "Save the skopeo inspect output to a log file"
+  copy:
+    content: "{{ skopeo_inspect_result.stdout }}"
+    dest: "{{ work_dir }}/bundle-skopeo-inspect.json"
+  when:
+    - skopeo_inspect_result is defined
+    - skopeo_inspect_result.stdout != ""
+
+- set_fact:
+    skopeo_inspect_json: "{{ skopeo_inspect_result.stdout | from_json }}"
+
+- set_fact:
+    is_bundle_image: "{{ skopeo_inspect_json.Labels['com.redhat.delivery.operator.bundle'] }}"
+    package_name: "{{ skopeo_inspect_json.Labels['operators.operatorframework.io.bundle.package.v1'] }}"
+    default_channel: "{{ skopeo_inspect_json.Labels['operators.operatorframework.io.bundle.channel.default.v1'] }}"
+    current_channel: "{{ skopeo_inspect_json.Labels['operators.operatorframework.io.bundle.channel.default.v1'] }}" # using default channel since the bundle only has one
+    ocp_versions: "{{ skopeo_inspect_json.Labels['com.redhat.openshift.versions'].split(',') }}"
+    is_backport: false
+
+- set_fact:
+    is_backport: "{{ skopeo_inspect_json.Labels['com.redhat.delivery.backport'] }}"
+  when: skopeo_inspect_json.Labels['com.redhat.delivery.backport'] is defined
+
+- name: "Determine the clusterserviceversion (CSV) path in the operator metadata directory"
+  shell: "egrep -lR \"kind:\\s*?ClusterServiceVersion*?\" {{ operator_work_dir }}/manifests/*"
+  register: csv_path_result
+
+- name: "Determine the directory of the current clusterserviceversion"
+  shell: "dirname {{ csv_path_result.stdout }}"
+  register: current_csv_dir_result
+
+- set_fact:
+    csv_path: "{{ csv_path_result.stdout }}"
+    current_csv_dir: "{{ current_csv_dir_result.stdout }}"
+
+- name: "Determine the current clusterserviceversion name"
+  shell: "{{ yq_bin_path }} r {{ csv_path }} 'metadata.name'"
```

Comment:
Could we use `{{ yq_bin_path }} read` instead of single letter?
It is just a bit longer.

Reply:
and the same for other places in the file